### PR TITLE
chore(*): Fixes #94

### LINKED ISSUE #94: Using gopsutil in a container
So there is a default /proc file system within a container that is used for container level pids etc... It would be nice to override where gopsutil looks so you could bind mount the host /proc file system into the container.

### PR DESCRIPTION
**Pull request by jchauncey**
Added the ability to fetch an alternative location for /proc via an
environment variable. If the env var is not set it will return /proc as
the default value.

### CODE REVIEW

File: process/types_darwin.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ package process
 
 #include <sys/sysctl.h>
 #include <sys/ucred.h>
-#include <sys/proc.h>
+#include <sys/proc).h>
```
Comment (shirou) [inline process/types_darwin.go]: Is this something typo?

Comment (jchauncey) [inline process/types_darwin.go]: Yup I'll fix it. Search and replace error

**Comment by jchauncey** [comment]
changed dfault in comment to default (but kept argument the same since default is a keyword). Also fixed the typo in `types_darwin.go`.

**Comment by shirou** [comment]
Thank you!

**Pull request merged**